src/EnergyPlus/BaseboardRadiator.cc

```diff
@@ -524,10 +523,7 @@ namespace BaseboardRadiator {
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
         int WaterInletNode;
         int ZoneNode;
-        static bool MyOneTimeFlag(true);
```

Comment:
Moved some flags to the member struct so we didn't need to allocate anything in here, and we didn't need a static in here anymore.